### PR TITLE
PLEASE IGNORE  MTV 465 doc sup ver hvider1

### PR DESCRIPTION
**Pull request by HagayVider1**
@ahadas  @mavital

MTV-476: https://issues.redhat.com/browse/MTV-476
MTV-465: https://issues.redhat.com/browse/MTV-465

Updated common-attributes.adoc and compatibility-guidelines.adoc files for MTV version 2.4.

### CODE REVIEW

**Comment by anarnold97** [comment]
@HagayVider1 - apart from the missing OpenStack version number, looks OK to me

File: documentation/modules/compatibility-guidelines.adoc [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 You must install compatible software versions.
 
-[cols="1,1,1,1,1", options="header"]
+[cols="1,1,1,1,1,1", options="header"]
 .Compatible software versions
 |===
-|{project-full} |{ocp} |{virt} |VMware vSphere |{rhv-full}
-|2.3.3 |4.10 or later |4.10 or later |6.5 or later |4.4.9 or later
+|{project-full} |{ocp} |{virt} |VMware vSphere |{rhv-full}|OpenStack
```
[the remark this answers is not in the document]
Comment (anarnold97) [reply documentation/modules/compatibility-guidelines.adoc:14]: Qin says - **we tested migration from OpenStack 16.1 to MTV 2.4.0, but didn't test versions below. I think the compatible version could be 16.1+**

@ahadas  WDYT?

File: documentation/modules/compatibility-guidelines.adoc [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
+|2.3.3 |4.10 or later |4.10 or later |6.5 or later |4.4.9 or later|16.1 or later
```
Comment (anarnold97) [inline documentation/modules/compatibility-guidelines.adoc:15]: @ahadas @HagayVider1 - I have added the OpenStack version as suggested by Qin, are you good with this?

File: documentation/modules/compatibility-guidelines.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahadas) [reply documentation/modules/compatibility-guidelines.adoc:14]: @anarnold97 yes, we shouldn't include older versions that were not tested so 16.1+ is alright

**Comment by anarnold97** [comment]
Closed obsolete

**Pull request closed without merging**